controllers/clustermaster_controller.go

```diff
@@ -148,12 +148,3 @@ func (r *ClusterMasterReconciler) SetupWithManager(mgr ctrl.Manager) error {
 		}).
 		Complete(r)
 }
-
-// recordInstrumentionData Record api profiling information to prometheus
-func recordInstrumentionData(start time.Time, req ctrl.Request, module string, name string) {
-	metricLabels := getPrometheusLabels(req, name)
-	metricLabels[labelModuleName] = module
-	metricLabels[labelMethodName] = name
-	value := float64(time.Since(start) / time.Millisecond)
-	apiTotalTimeMetricEvents.With(metricLabels).Set(value)
-}
```

Comment:
Any reason why this is removed?

Reply:
This function is now shared between both Master and Manager CRDs. It's not bound to the type so we only need one declaration. I chose to move this to the `clustermanager_controller.go` to avoid it being removed when we deprecate the master.